packages/gatsby-kontent-components/tests/rich-text-emelemt.spec.tsx

```diff
@@ -1,92 +1,135 @@
-    "width": 500
-  }
+    description: null,
+    height: 500,
+    image_id: 'd32b8ad5-0cf4-47a8-8b53-ed4a1e80dc88',
+    url:
+      'https://assets-us-01.kc-usercontent.com:443/0fe3ab32-97a8-005d-6928-eda983ea70a5/44299668-b37b-4224-a115-1fd66f7d7b36/Yprofile.jpg',
+    width: 500,
+  },
 ];
 const linkedItems = [
   {
-    "__typename": "kontent_item_website",
-    "system": {
-      "codename": "ondrej_chrastina_tech"
+    __typename: 'kontent_item_website',
+    system: {
+      codename: 'ondrej_chrastina_tech',
     },
-    "elements": {
-      "name": {
-        "value": "ondrej.chrastina.tech"
+    elements: {
+      name: {
+        value: 'ondrej.chrastina.tech',
       },
-      "slug": {
-        "value": "ondrej-chrastina-tech"
+      slug: {
+        value: 'ondrej-chrastina-tech',
       },
-      "summary": {
-        "value": "Ondřej Chrastina's personal site"
+      summary: {
+        value: "Ondřej Chrastina's personal site",
       },
-      "url": {
-        "value": "https://ondrej.chrastina.tech"
+      url: {
+        value: 'https://ondrej.chrastina.tech',
       },
-      "source_repository": {
-        "value": [
+      source_repository: {
+        value: [
           {
-            "__typename": "kontent_item_repository",
-            "elements": {
-              "slug": {
-                "value": "simply007-simply007-github-io"
-              }
-            }
-          }
-        ]
-      }
-    }
+            __typename: 'kontent_item_repository',
+            elements: {
+              slug: {
+                value: 'simply007-simply007-github-io',
+              },
+            },
+          },
+        ],
+      },
+    },
   },
   {
-    "__typename": "kontent_item_repository",
-    "system": {
-      "codename": "simply007_kontent_gatsby_benchmark"
+    __typename: 'kontent_item_repository',
+    system: {
+      codename: 'simply007_kontent_gatsby_benchmark',
     },
-    "elements": {
-      "name": {
-        "value": "Simply007/kontent-gatsby-benchmark"
+    elements: {
+      name: {
+        value: 'Simply007/kontent-gatsby-benchmark',
       },
-      "slug": {
-        "value": "simply007-kontent-gatsby-benchmark"
+      slug: {
+        value: 'simply007-kontent-gatsby-benchmark',
       },
-      "summary": {
-        "value": "Example site for Kontent as a source for \"Will It Build\". Should/Will be generalized into e.g. a theme."
+      summary: {
+        value:
+          'Example site for Kontent as a source for "Will It Build". Should/Will be generalized into e.g. a theme.',
       },
-      "url": {
-        "value": "https://github.com/Simply007/kontent-gatsby-benchmark"
-      }
-    }
-  }
+      url: {
+        value: 'https://github.com/Simply007/kontent-gatsby-benchmark',
+      },
+    },
+  },
 ];
 
-describe("<RichTextElement/>", () => {
+describe('<ImageElement />', () => {

```

Comment:
I would split this test out of the rich text because the component is separated as well.

Reply:
Fixed in https://github.com/Kentico/kontent-gatsby-packages/pull/174/commits/e7eef3a40950a78e8cf795455e38b59e5f41b0a9